Documentation/Helm-Charts/ceph-cluster-chart.gotmpl.md

```diff
@@ -41,14 +41,16 @@ Rook currently publishes builds of this chart to the `release` and `master` chan
 The release channel is the most recent release of Rook that is considered stable for the community.
 
 The example install assumes you have first installed the [Rook Operator Helm Chart](operator-chart.md)
-and created your customized values-override.yaml.
+and created your customized values.yaml.
 
 ```console
 helm repo add rook-release https://charts.rook.io/release
 helm install --create-namespace --namespace rook-ceph rook-ceph-cluster \
-   --set operatorNamespace=rook-ceph rook-release/rook-ceph-cluster -f values-override.yaml
+   --set operatorNamespace=rook-ceph rook-release/rook-ceph-cluster -f values.yaml
 ```
-
+!!! Note
+    --namespace specify the cephcluster namespace.
+    operatorNamespace should be same as where the operator is created.
```

Comment:
I'd suggest no need for this, it is hopefully self-explanatory.
```suggestion

```

Reply:
okay